modules/ROOT/pages/migration-transports.adoc

```diff
@@ -63,7 +60,7 @@ provides detail on the exact properties that the connector requires.
 == Service Overrides
 
 In Mule 4, service override customizations through custom Java code are not
```

Comment:
Line 62 services? I don't know. There is something wrong there.

Reply:
No, it's 'the ability to customize Mule to override a service' or similar, leaving for @dukesphere as I'm fuzzy on what this means too.